src/components/dashboard/LogList.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LogRow from './LogRow';
```

Comment:
Good to have a new line between installed 3rd party package and our imported Components .